src/domain/user/layout/UserTabs.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { buildUserProfileSettingsUrl, buildUserProfileUrl } from '../../../common/utils/urlBuilders';
+import HeaderNavigationTab from '../../shared/components/PageHeader/HeaderNavigationTab';
+import { useTranslation } from 'react-i18next';
+import HeaderNavigationTabs from '../../shared/components/PageHeader/HeaderNavigationTabs';
+import { EntityTabsProps } from '../../shared/layout/PageLayout/EntityPageLayout';
+import { useUserContext } from '../hooks/useUserContext';
+import { useUserMetadata } from '../hooks/useUserMetadata';
+import { useUrlParams } from '../../../hooks';
+
+const routes = {
+  profile: 'profile',
+  settings: 'settings',
+};
+
+const UserTabs = (props: EntityTabsProps) => {
+  const { t } = useTranslation();
+  const { user, loading: loadingUserContext } = useUserContext();
+
+  const { userNameId = '' } = useUrlParams();
+  const { user: userMetadata, loading: loadingUserMetadata } = useUserMetadata(userNameId);
+  const isCurrentUser =
+    (!loadingUserContext && !loadingUserMetadata && user?.user.id === userMetadata?.user.id) || false;
```

Comment:
the first expression already resolves to boolean - it doesn't need to be defaulted to false.